tests/test_integration_all_rust.py

```diff
@@ -630,6 +679,9 @@ def setup_connection_server(connection_binary):
     else:
         write_config_to_env(CONNECTION_CONFIG, CONNECTION_SETTINGS_PREFIX)
     cmd = [connection_binary]
+    if BT_PROCESS is not None:
+        cmd.extend(["--features", "emulator"])
```

Comment:
```suggestion
```
and below -- so I believe these args aren't actually used because `shell=True` is specified. Regardless these are solely build time options -- actually running them with these arguments should be resulting in an error.